test/__snapshots__/Autocomplete.test.tsx.snap

```diff
@@ -15,9 +15,10 @@ Object {
             class="MuiFormControl-root MuiFormControl-marginNormal MuiFormControl-fullWidth MuiTextField-root css-17vbkzs-MuiFormControl-root-MuiTextField-root"
           >
             <label
-              class="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined MuiFormLabel-root MuiFormLabel-colorPrimary Mui-required css-14s5rfu-MuiFormLabel-root-MuiInputLabel-root"
-              data-shrink="false"
-              id="undefined-label"
+              class="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-outlined MuiFormLabel-root MuiFormLabel-colorPrimary MuiFormLabel-filled Mui-required css-1sumxir-MuiFormLabel-root-MuiInputLabel-root"
+              data-shrink="true"
```

Comment:
keep an eye out for changes like this in the snapshots where something that was false becomes true. Not sure if it was an original error in the snap or not though.

Reply:
any idea what data-shrink is?

Reply:
inputLabel has a shrink property. It at least used to be an element within autocomplete.

https://mui.com/material-ui/api/input-label/